openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/teams.less

```diff
@@ -62,7 +62,10 @@
     .ant-table-container .ant-table-content table tbody {
       .ant-table-row-level-1,
       .ant-table-row-level-2,
-      .ant-table-row-level-3 {
+      .ant-table-row-level-3,
+      .ant-table-row-level-4,
+      .ant-table-row-level-5,
```

Comment:
Can you explain this class?
we might use `ant-table-row-level-*`?

Reply:
@chirag-madlani  updated the css for the parent and child row!